src/openlcb/TractionTrain.hxx

```diff
@@ -146,7 +162,15 @@ private:
 class TrainNodeWithConsist : public TrainNode {
 public:
     ~TrainNodeWithConsist();
-    
+
+    bool function_policy(NodeHandle src, uint8_t command_byte, uint32_t fnum,
+        uint16_t value, Notifiable *done) override
+    {
+        AutoNotify an(done);
+        // The default function policy applies everything.
```

Comment:
This comment should be the doxygen header comment for the function.

Reply:
done